healthx/handler_test.go

```diff
@@ -47,9 +65,10 @@ func TestHealth(t *testing.T) {
 			},
 		},
 	}
+
 	router := httprouter.New()
-	handler.SetHealthRoutes(router, true)
-	handler.SetVersionRoutes(router)
+	handler.SetHealthRoutes(router, true, loggerMiddleware)
+	handler.SetVersionRoutes(router, loggerMiddleware)
```

Comment:
It would be good to add some assertions here instead of logging. Logs are not shown in tests without `-v`, and not checked by automation.

Reply:
these were just to satisfy the tests for the above change, in case the above changes wouldn't get approved.
Will add needed assertions with in the next commit